app/public/views/periods/table/periodsTableHeader.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import { getHeaderSpecial, headerSpecPresent, nonDisplayable } from '../../userView/data/headersSpecials.js';
+
+export default function periodsTableHeader(pageName, visibleFields, data, dataAccessModel) {
```

Comment:
`dataAccessModel` is used for getting `headerSpecials` and is needed for retro-compatibility (handling navigation to other views that use the old model)